composer.json

```
@@ -5,8 +5,9 @@
     "license": [
         "AGPL-3.0+"
     ],
+    "version": "2.1.4"
     "require": {
-        "php": "~7.0.0|~7.1.0|~7.2.0|~7.3.0|~7.4.0|~8.1.0|~8.2.0",
+        "php": "~7.0.0|~7.1.0|~7.2.0|~7.3.0|~7.4.0|~8.1.0|~8.2.0|~8.3",
```

This can be shortened to:

```suggestion
        "php": "^7.0 || ^8.1",
```

The single pipe `|` notation [is deprecated](https://getcomposer.org/doc/articles/versions.md#version-range).